src/lib/components/input/Input.component.js

```diff
@@ -37,8 +39,44 @@ const InputRenderer = ({
   checked,
   onChange,
   options = [],
+  min,
+  max,
   ...rest
 }) => {
+  const inputEl = useRef(null);
+
+  const handleNumberClick = (e) => {
+    /*
+     ** Since react is using Synthetic event we have to do the following to be
+     ** able to programmatically dispatch the onChange event from the input
+     */
+    const valuePropDescriptor = Object.getOwnPropertyDescriptor(
+      window.HTMLInputElement.prototype,
+      "value",
+    );
+    if (valuePropDescriptor && inputEl.current) {
+      const nativeInputValueSetter = valuePropDescriptor.set;
+      if (nativeInputValueSetter) {
+        if (e.target.dataset.core === "up")
+          nativeInputValueSetter.call(
+            inputEl.current,
+            max && parseInt(value) + 1 <= parseInt(max)
```

Comment:
if I don't specify `max`, I may not be able to increment?

Reply:
Indeed... fixing